opsboy

```diff
@@ -306,7 +306,9 @@ sub make ($$) {
             my $dir = shift @args;
             my $branch = shift @args;
 
-            if (!defined $branch) {
+            if (defined $branch) {
+                $branch =~ s/\$(\w+)/$ENV{$1}/g;
```

Comment:
Changing the regex to `\$[a-zA-Z]\w*` is better.

Reply:
Updated